ee/clickhouse/sql/person.py

```diff
@@ -316,7 +316,9 @@
 AND person_id IN
 (
     SELECT id
-    FROM person
+    FROM (
+        SELECT id, argMax(properties, person._timestamp) as properties FROM person WHERE team_id = %(team_id)s GROUP BY id
+    )
     WHERE team_id = %(team_id)s {filters}
```

Comment:
need to make sure we only use properties column from this table when adding filters